.pre-commit-config.yaml

```diff
@@ -95,7 +90,7 @@ repos:
   rev: 1.13.0
```

Comment:
In line with your earlier comment about the trailing zeros, should rev: on ln: 90 and additional_depencies: ln: 93, also have their zeros truncated?

Reply:
Good eye!  The tool that automatically updates pre-commit hooks pins them to specific versions for stability, so the zeros should be there.  Thanks!